src/microbe_stage/components/SiderophoreProjectile.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Components;
+
+using DefaultEcs;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Defines toxin damage dealt by an entity
+/// </summary>
+[JSONDynamicTypeAllowed]
+public struct SiderophoreProjectile
+{
+    /// <summary>
+    ///   Scales the efficiency
+    /// </summary>
+    public float Amount;
+
+    /// <summary>
+    ///   Scales the efficiency
+    /// </summary>
+    public bool IsUsed;
+
+    /// <summary>
+    ///   Sender
+    /// </summary>
+    public Entity Sender;
```

Comment:
Due to struct padding, largest fields should be first. So `Sender` should be first, then the float fields and finally all the booleans.